src/main/java/org/sonatype/repository/conan/internal/metadata/ConanAbsoluteUrlRemover.java

```diff
@@ -1,76 +0,0 @@
-/*
- * Sonatype Nexus (TM) Open Source Version
- * Copyright (c) 2017-present Sonatype, Inc.
- * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
- *
- * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
- * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
- *
- * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
- * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
- * Eclipse Foundation. All other trademarks are the property of their respective owners.
- */
-package org.sonatype.repository.conan.internal.metadata;
-
-import java.io.BufferedWriter;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.file.Path;
-
-import javax.annotation.Nullable;
-
-import org.sonatype.goodies.common.ComponentSupport;
-import org.sonatype.nexus.repository.Repository;
-import org.sonatype.nexus.repository.storage.StorageFacet;
-import org.sonatype.nexus.repository.storage.TempBlob;
-
-import org.apache.commons.io.IOUtils;
-
-import static java.nio.file.Files.createTempFile;
-import static java.nio.file.Files.delete;
-import static java.nio.file.Files.newInputStream;
-import static java.util.UUID.randomUUID;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyHelper.HASH_ALGORITHMS;
-
-/**
- * download_url files contain absolute paths to each asset
- *
- * This class removes the absolute address so as to redirect back to this repository
- *
- * @since 3.conan
- */
-public class ConanAbsoluteUrlRemover
```

Comment:
Allowing Nexus to hit multiple remotes is a different way of operating compared to other formats, and I could understand if some users weren't comfortable with that. I think it might be nice to make it configurable at the repository level. "Absolute URL handling" and then have options "Remove: this could cause builds to fail" and "Replace: this will redirect through NXRM but Nexus will go to URLs that aren't the configured remote" and then make "Replace" the default.

Reply:
I think I would prefer to do this work as a separate PR.
However just to be clear. There are two levels of redirection. This PR is changing the metadata reference locations only. A further PR will be to alter the python files to alter the source locations if building from source is required. Does this mean that we will need some kind of multiple selection? Or keep it simple, all or nothing.

Reply:
That sounds good to me and yep all or nothing, let's not over complicate it just yet 😄